pkg/fanal/analyzer/const.go

```diff
@@ -128,4 +128,10 @@ var (
 
 	// TypeConfigFiles has all config file analyzers
 	TypeConfigFiles = []Type{TypeYaml, TypeJSON, TypeDockerfile, TypeTerraform, TypeCloudFormation, TypeHelm}
+
+	// TypeVulnFilesWithoutLicenseInfo has all language and lock file analyzers without files which contains information about licenses
+	TypeVulnFilesWithoutLicenseInfo = []Type{TypeRpm, TypeRpmqa,
```

Comment:
I think it is hard to maintain the list of these analyzers. Actually, RPM and APK have license information. We may add support for license in JAR files, etc.